src/libLookup/Lookup.cpp

```diff
@@ -5581,11 +5581,6 @@ bool Lookup::ProcessForwardTxn(const bytes& message, unsigned int offset,
   if (!ARCHIVAL_LOOKUP) {
     uint32_t shard_size = m_mediator.m_ds->GetNumShards();
 
-    if (shard_size == 0) {
-      LOG_GENERAL(WARNING, "Shard size 0");
-      return false;
```

Comment:
should u return?

Reply:
i think thats the purpose, the transaction needs to be forwarded to shard 0 ( ds-shard in above case ).
should not return right? if i got you correctly..